src/components/SiteHeader/SiteHeader.tsx

```diff
@@ -24,39 +26,43 @@ const Headers: Header[] = [
   { href: '/about', title: 'about' },
 ]
 
-const SiteHeader = (props: Props) => {
+const SiteHeader = (props: { location: GatsbyLocation }) => {
   const pathname: string = _.get(props, 'location.pathname', '/')
   const isRoot: boolean = _.eq(pathname, '/')
+
+  const theme: HeaderTheme = _.startsWith(pathname, '/photography')
+    ? HeaderTheme.LIGHT
+    : HeaderTheme.DARK
+
   return (
-    <Headroom
-      className="bg-gray-primary"
-      style={{
-        MozTransition: 'all 0.7s ease-in-out',
-        OTransition: 'all 0.7s ease-in-out',
-        WebkitTransition: 'all 0.7s ease-in-out',
-        transition: 'all 0.7s ease-in-out',
-      }}
-    >
-      <header
+    <Headroom>
+      <Heading
+        as="header"
         id="site-header"
-        className={`b--hot-pink bb bg-gray-primary center flex pb1 site-header w-100 ${
-          !isRoot ? 'bw1' : ''
-        }`}
+        alignItems="center"
+        fontFamily="body"
+        className={classNames('bb flex justify-center', {
+          'bg-gray-primary b--hot-pink': theme === HeaderTheme.DARK,
+          'bg-near-white b--black-10': theme === HeaderTheme.LIGHT,
+          bw1: !isRoot,
+        })}
+        style={{
+          transition: 'all 0.5s ease-in-out',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 20 spaces but found 10. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)